pkgs/top-level/qt5-packages.nix

```diff
@@ -203,6 +203,8 @@ in (kdeFrameworks // plasmaMobileGear // plasma5 // plasma5.thirdParty // kdeGea
 
   qtforkawesome = callPackage ../development/libraries/qtforkawesome { };
 
+  qtgreet = callPackage ../os-specific/linux/qtgreet/default.nix { };
```

Comment:
Why should this be put as os-specific?

`os-specific` is mostly for low-level, "unportable" and highly specific things strongly tied to a certain environment. Things like alsamixer, systemd, kernel modules, Apple SDK...

The things you are trying to package here are highly dependent on ... wayland.

And, well, someone can argue that Wayland runs only on Linux, but this is mostly by lack of manpower. There is nothing inherently OS specific on Wayland and its apps/APIs.

Indeed there is some incipient support going on NetBSD: https://blog.netbsd.org/tnf/entry/wayland_on_netbsd_trials_and

Reply:
Because that's where all the other greeters are too. (See the change in all-packages.nix.)

I don't have any other systems I could test this on. I can't maintain the package for them. It'd be disingenuous to claim they're supported.  
If someone comes around and validates the package on other systems, moving the files and changing supported platforms are trivial changes.

Reply:
> Because that's where all the other greeters are too. (See the change in all-packages.nix.)

Good to know. I will move them for a more appropriate place.

> I don't have any other systems I could test this on. I can't maintain the package for them.

You are not understanding the point here. 

The `os-specific` directory should have only softwares strongly tied to an operating system.

This is not a folder for _I don't have a RISC-V 128 bits running NetBSD_. The _I don't have a RISC-V 128 bits running NetBSD_ info is meant to be put on `meta.platforms`.

> If someone comes around and validates the package on other systems, moving the files and changing supported platforms are trivial changes.

Ok then.

Reply:
> Good to know. I will move them for a more appropriate place.

Great! I support that. But I think it's better to follow convention now and handle changing the convention in another PR.

> You are not understanding the point here.

Sorry if I wasn't clear, this is in response to `meta.platforms`, not the file location.